cpp/src/fil/fil.h

```diff
@@ -98,6 +111,15 @@ struct forest_params_t {
 void init_dense(const cumlHandle& h, forest_t* pf,
```

Comment:
Please can you change the variable names from alphabets to whole words for all the functions in the .cu and .h file

Reply:
Done for functions introduced in this pull request.